src/clusterfuzz/_internal/base/tasks/__init__.py

```diff
@@ -616,6 +656,9 @@ def run(self):
         if time_left <= 0:
           logs.info('Lease reached maximum lease time of {} seconds, '
                     'stopping renewal.'.format(self._max_lease_seconds))
+          if self._ack_on_timeout:
+            logs.info('Acking on timeout')
+            self._message.ack()
```

Comment:
Why acking? Why is this preferred to exiting and retrying later?

Reply:
For fuzz tasks, I'd rather things starve for now than gum up the queue.